lib/commands/create.dart

```diff
@@ -30,6 +30,12 @@ class TizenCreateCommand extends CreateCommand {
       negatable: false,
       help: 'Create a Tizen service application.',
     );
+    argParser.addFlag(
+      'tizen-multi',
```

Comment:
How about
```dart
argParser.addOption(
  'app-type',
  defaultsTo: 'ui',
  allowed: <String>['ui', 'service', 'multi'],
);
```

Reply:
I agree, that your suggestion is very good, but I created new issue about this idea - https://github.com/flutter-tizen/flutter-tizen/issues/207
I will take care about #207 as soon as this base will be ready.